<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startBt">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>750</y>
      <width>83</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QLabel" name="Screen">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>460</width>
      <height>290</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>SCREEN</string>
    </property>
   </widget>
   <widget class="QComboBox" name="color_combobx">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Random Color</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fixed Color</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Color Preferences Menu</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Animation Preferences Menu</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Circles</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Stars</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Squares</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SnowFlakes</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="zenopix_icon">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>hppy</string>
    </property>
   </widget>
   <widget class="QLabel" name="znpx">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>0</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>znpx</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>750</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>üå®‚ùÑùôÉùôñùô•ùô•ùôÆ ùôâùôöùô¨ ùôîùôöùôñùôß‚ùÑüå®Ô∏è</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>210</y>
      <width>231</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>210</y>
      <width>81</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>icon</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="NumberOfStar">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>50</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Lower Radius Limit</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="lowerR_spnbx">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="UpperR_spnbx">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>3</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Upper Radius Limit</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="LowerT_spnbx">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1000</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="UpperT_spnbx">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>5000</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Lower onTime</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Upper onTime</string>
    </property>
   </widget>
   <widget class="QSlider" name="Rising_sldr">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>361</width>
      <height>18</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Rising Speed</string>
    </property>
   </widget>
   <widget class="QSlider" name="falling_sldr">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>361</width>
      <height>18</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Falling Speed</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Number Of Shapes</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>522</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
